src/Microsoft.AspNet.Html.Abstractions/IHtmlContentBuilder.cs

```diff
@@ -29,7 +29,7 @@ public interface IHtmlContentBuilder : IHtmlContent
         /// </summary>
         /// <param name="encoded">The HTML encoded <see cref="string"/> to append.</param>
         /// <returns>The <see cref="IHtmlContentBuilder"/>.</returns>
-        IHtmlContentBuilder AppendEncoded(string encoded);
+        IHtmlContentBuilder AppendHtml(string encoded);
```

Comment:
The parameter name is entirely correct and a good counterpoint to the `unencoded` parameter in `Append()`. In your example `"<b>"` is as encoded as it's going to get.